tck/src/main/java/org/eclipse/microprofile/config/tck/CDIPlainInjectionTest.java

```diff
@@ -104,36 +106,34 @@ public void canInjectSimpleValuesWhenDefined() {
-                ConfigProvider.getConfig().getValue("my.string.property", String.class))));
-        assertThat(bean.booleanProperty, is(equalTo(
-                ConfigProvider.getConfig().getValue("my.boolean.property", Boolean.class))));
-        assertThat(bean.byteProperty, is(equalTo(
-                ConfigProvider.getConfig().getValue("my.byte.property", Byte.class))));
-        assertThat(bean.shortProperty, is(equalTo(
-                ConfigProvider.getConfig().getValue("my.short.property", Short.class))));
-        assertThat(bean.intProperty, is(equalTo(
-                ConfigProvider.getConfig().getValue("my.int.property", Integer.class))));
-        assertThat(bean.longProperty, is(equalTo(
-                ConfigProvider.getConfig().getValue("my.long.property", Long.class))));
-        assertThat(bean.floatProperty, is(floatCloseTo(
-                ConfigProvider.getConfig().getValue("my.float.property", Float.class), 0.1f)));
-        assertThat(bean.doubleProperty, is(closeTo(
-                ConfigProvider.getConfig().getValue("my.double.property", Double.class), 0.1)));
-        assertThat(bean.charProperty, is(equalTo(
-                ConfigProvider.getConfig().getValue("my.char.property", Character.class))));
-
-        assertThat(bean.doublePropertyWithDefaultValue, is(closeTo(
-                ConfigProvider.getConfig().getOptionalValue("my.not.configured.double.property", Double.class)
-                        .orElse(3.1415), 0.1)));
+        assertThat(bean.stringProperty,
+                is(equalTo(ConfigProvider.getConfig().getValue("my.string.property", String.class))));
+        assertThat(bean.booleanProperty,
+                is(equalTo(ConfigProvider.getConfig().getValue("my.boolean.property", Boolean.class))));
+        assertThat(bean.byteProperty, is(equalTo(ConfigProvider.getConfig().getValue("my.byte.property", Byte.class))));
+        assertThat(bean.shortProperty,
+                is(equalTo(ConfigProvider.getConfig().getValue("my.short.property", Short.class))));
+        assertThat(bean.intProperty,
+                is(equalTo(ConfigProvider.getConfig().getValue("my.int.property", Integer.class))));
+        assertThat(bean.longProperty, is(equalTo(ConfigProvider.getConfig().getValue("my.long.property", Long.class))));
+        assertThat(bean.floatProperty,
+                is(floatCloseTo(ConfigProvider.getConfig().getValue("my.float.property", Float.class), 0.1f)));
+        assertThat(bean.doubleProperty,
+                is(closeTo(ConfigProvider.getConfig().getValue("my.double.property", Double.class), 0.1)));
+        assertThat(bean.charProperty,
+                is(equalTo(ConfigProvider.getConfig().getValue("my.char.property", Character.class))));
+
+        assertThat(bean.doublePropertyWithDefaultValue, is(closeTo(ConfigProvider.getConfig()
+                .getOptionalValue("my.not.configured.double.property", Double.class).orElse(3.1415), 0.1)));
```

Comment:
What is the actual change here?  Was this just reformatted, and if so could it be reverted?

Reply:
For some reason, my reformatting tool kicked in. Cleaned up now.

Reply:
Maybe we could adopt https://github.com/revelc/formatter-maven-plugin? Actually I'll submit an issue with it.